### PR TITLE
LPS-129904

### PR DESCRIPTION
**Pull request by mlan0**
Related Issue: https://issues.liferay.com/browse/LPS-129904

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java [hunk 1]
```diff
@@ -684,7 +684,7 @@ public JSONObject getFormLocalizedDescriptionJSONObject()
 		DDMFormInstance formInstance = getDDMFormInstance();
 
 		if (formInstance == null) {
-			jsonObject.put(getDefaultLanguageId(), "");
+			jsonObject.put(getDefaultLanguageId(), getFormDescription());
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java:687]: This fix seems to be done by surgeons :man_health_worker:, boys! Good job! :wink:

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**